pandas/core/frame.py

```diff
@@ -4828,10 +4829,11 @@ def apply(self, func, axis=0, broadcast=None, raw=False, reduce=None,
         Parameters
         ----------
         func : function
-            Function to apply to each column/row
+            Function to apply to each column/row.
         axis : {0 or 'index', 1 or 'columns'}, default 0
+            Axis along which the function is applied
             * 0 or 'index': apply function to each column
-            * 1 or 'columns': apply function to each row
+            * 1 or 'columns': apply function to each row.
```

Comment:
These bullets should both have periods, or not. I don't really have a preference, but probably best to add a period to both.